docs/source/howto/installation.rst

```diff
@@ -18,7 +18,149 @@ Managing profiles
+This file is shell specific, but likely one of the following:
+
+    * the startup file of your shell (``.bashrc``, ``.zsh``, ...), if aiida is installed system-wide
+    * the `activation script <https://virtualenv.pypa.io/en/latest/userguide/#activate-script>`_ of your virtual environment
+    * a `startup file <https://conda.io/docs/user-guide/tasks/manage-environments.html#saving-environment-variables>`_ for your conda environment
+
+
+.. important::
+
+    After you have added the line to the start up script, make sure to restart the terminal or source the script for the changes to take effect
+
+
+.. _how-to:installation:configure:options:
+
+Configuring profile options
+---------------------------
+In addition to the configuration of AiiDA instances described in the previous sections, AiiDA also provides various configurational options for profiles.
+To control the configuration options of profiles use the :ref:`verdi config<reference:command-line:verdi-config>` command.
+To set a configurational option, simply pass the name of the option and the value to set ``verdi config OPTION_NAME OPTION_VALUE``.
+The available options are tab-completed, so simply type ``verdi config`` and thit <TAB> twice to list them.
+
+For example, if you want to change the default number of workers that are created when you start the daemon, you can run::
+
+.. code:: bash
+
+    (aiida): verdi config daemon.default_workers 5
+    Success: daemon.default_workers set to 5 for profile-one
+
+You can check the currently defined value of any option by simply calling the command without specifying a value, for example:
+
+.. code:: bash
+
+    (aiida): verdi config daemon.default_workers
+    5
+
+If no value is displayed, it means that no value has ever explicitly been set for this particular option and the default will always be used.
+By default any option set through ``verdi config`` will be applied to the current default profile.
+To change the profile you can use the :ref:`profile option<topics:cli:profile>`.
+
+To undo the configuration of a particular option and reset it so the default value is used, you can use the ``--unset`` option:
+
+.. code:: bash
+
+    (aiida): verdi config daemon.default_workers --unset
+    Success: daemon.default_workers unset for profile-one
+
+If you want to set a particular option that should be applied to all profiles, you can use the ``--global`` flag:
+
+.. code:: bash
+
+    (aiida): verdi config daemon.default_workers 5 --global
+    Success: daemon.default_workers set to 5 globally
+
+and just as on a per-profile basis, this can be undone with the ``--unset`` flag:
+
+.. code:: bash
+
+    (aiida): verdi config daemon.default_workers --unset --global
+    Success: daemon.default_workers unset globally
+
+.. important::
+
+    Remember that if you change options that affect the daemon as well (e.g. the `logging.aiida_loglevel``), the changes will not take effect until *after* you have restarted the daemon.
```

Comment:
```suggestion
    Changes that affect the daemon (e.g. ``logging.aiida_loglevel``) will only take affect after restarting the daemon.
```
It preferable to use affirmative language for these kinds of notes, see [here](https://devguide.python.org/documenting/#affirmative-tone) for an explanation.

Reply:
Thanks for the link, very useful. Will adapt it